src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
+    /*****
```

Comment:
let's make it as it was before :)
```suggestion
    /**
```